tests/test_core/test_db_containers.py

```diff
@@ -56,7 +56,7 @@ def test_docker_run_oracle():
 
 def test_docker_run_mssql():
     image = 'mcr.microsoft.com/azure-sql-edge'
-    dialect = 'mssql+pymssql'
+    dialect = 'mssql+pyodbc'
```

Comment:
This won't work, right? Because you haven't specified the driver?

Perhaps test manually and then revert the test changes for the merge

Reply:
Gonna test both dialects